skycultures/arabic/constellationship.fab

```diff
@@ -0,0 +1,242 @@
+﻿#0100 الفرقدان

```

Comment:
Please use the English names in the comments.

Reply:
Ok, done.